desktop/mac/release-notes.md

```diff
@@ -12,14 +12,34 @@ toc_max: 2
 
 This page contains information about the new features, improvements, known issues, and bug fixes in Docker Desktop releases.
 
+## Docker Desktop 4.0.1
+2021-09-09
+
+> Download Docker Desktop
+>
+> [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
+> [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
+
+### Upgrades
+
+- [Compose V2 RC3](https://github.com/docker/compose-cli/releases/tag/v2.0.0-rc.3)
+  - Compose v2 is now hosted on github.com/docker/compose.
+  - Fixed go panic on downscale using `compose up --scale`.
+  - Fixed  a race condition in `compose run --rm` while capturing exit code.
+
+### Bug fixes and minor changes
+
+- Fixed a bug where copy-paste was not available in the Dashboard.
 
 ## Docker Desktop 4.0.0
 2021-08-31
 
 > Download Docker Desktop
 >
-> [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-amd64){: .button .primary-btn }
-> [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){: .button .primary-btn }
+> {%- include eula.md -%}
```

Comment:
perhaps the eula include as well 🤔

Reply:
oh I just copied pasted from the last part! 🤔

Reply:
Yes, the older releases did not have the "accept agreement" in the installer, which is why it was added in the docs. Now that all new binaries we ship have this, I think we can remove it for the new downloads, so if could be simplified to:


```markdown
> Download Docker Desktop
>
> [Mac with Intel chip](https://desktop.docker.com/mac/stable/amd64/67817/Docker.dmg) |
> [Mac with Apple chip](https://desktop.docker.com/mac/stable/arm64/67817/Docker.dmg)
```

Reply:
Yes, that makes sense. We don't need the EULA class for 4.0 and higher releases